### PR TITLE
Pass email lists on notifications

### LINKED ISSUE #2073: Airllock - Add notifications messages across the system
As a TRE Developer
I want to send notification events on different scenarios
So that the notification mechanism can process them

## Acceptance criteria

- [ ] A 'request is pending review' event is sent to the ws owners of the ws, when the airlock processor gets a status changes event saying the status is now 'in-review'
- [ ] A 'request status changed event'  is sent to the researchers of the ws, when the request changes status (FYI)

### PR DESCRIPTION
**Pull request by LizaShak**
# PR for issue

## What is being addressed

Fixes #2073 by passing email lists  on notifications

## How is this addressed

- Get role assignments from MS graph
- Batch requests to get users data
- Parse to email lists

### CODE REVIEW

File: api_app/main.py [hunk 1]
```diff
@@ -76,4 +76,4 @@ async def update_airlock_request_status() -> None:
     await receive_step_result_message_and_update_status(app)
 
 if __name__ == "__main__":
-    uvicorn.run(app, host="0.0.0.0", port=8000)
```
Comment (anatbal) [inline api_app/main.py:79]: This change should be fixed

**Review by eladiw — APPROVED** [review]

File: api_app/api/routes/airlock_resource_helpers.py [hunk 2]
```diff
@@ -35,15 +37,16 @@ async def save_and_publish_event_airlock_request(airlock_request: AirlockRequest
     try:
         logging.debug(f"Sending status changed event for airlock request item: {airlock_request.id}")
         await send_status_changed_event(airlock_request)
-        # TODO - replace with the emails list
-        await send_airlock_notification_event(airlock_request, [], [])
+        access_service = get_access_service()
+        role_assignment_details = access_service.get_workspace_role_assignment_details(workspace)
+        await send_airlock_notification_event(airlock_request, role_assignment_details["researcher_emails"], role_assignment_details["owner_emails"])
```
Comment (SharonHart) [inline api_app/api/routes/airlock_resource_helpers.py:42]: @LizaShak @anatbal you didn't define type hints for send_airlock_notification_event, but I suspect you are passing a list? 
(The logic app accepts a `;` delimited list)

**Pull request merged**